hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/parse/UnsupportedOperationVisitor.java

```diff
@@ -96,9 +96,11 @@
         SUPPORTED_KINDS.add(SqlKind.GREATER_THAN);
         SUPPORTED_KINDS.add(SqlKind.GREATER_THAN_OR_EQUAL);
         SUPPORTED_KINDS.add(SqlKind.LESS_THAN_OR_EQUAL);
+        SUPPORTED_KINDS.add(SqlKind.IN);
```

Comment:
in and between are now enabled, but looks like there are no tests targeting them specifically

Reply:
This is an incorrect merge from the `sql` branch. Removed support for `IN`/`BETWEEN`.